cmd/gobmp/gobmp.go

```diff
@@ -108,6 +117,28 @@ func main() {
 	// Starting Interceptor server
 	bmpSrv.Start()
 
+	httpPortStr, ok := os.LookupEnv("HTTP_PORT")
```

Comment:
I believe its better if the environment variable specifies `HEALTHCHECK`, like `HEALTHCHECK_HTTP_PORT`

In general, healthchecks are usually served by a separate HTTP server, on an internal port that is not exposed to the client services.
If a service serves HTTP traffic, this HTTP server should run on a different port and be exposed accordingly.